karton/core/backend.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, config):
             endpoint=config["minio"]["address"],
             access_key=config["minio"]["access_key"],
             secret_key=config["minio"]["secret_key"],
-            secure=bool(int(config["minio"].get("secure", True))),
+            secure=config.config.getboolean("minio", "secure", fallback=True),
```

Comment:
I had to push again, since in the previous change set it read `config.getboolean`.

Reply:
Yeah, `config.config` is quite cumbersome. I think it should be simplified it in the future.